src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/RuntimeBinderExtensions.cs

```diff
@@ -23,9 +23,9 @@ public static bool IsNullableType(this Type type)
         // or not members have been added to an AggSym or not.
         public static bool IsEquivalentTo(this MemberInfo mi1, MemberInfo mi2)
         {
-            if (mi1 == null || mi2 == null)
+            if (mi1 == null | mi2 == null)
             {
```

Comment:
Why change to eager operators?

Reply:
Darn. That was "thinking out loud at the keyboard" that got left in by mistake. (It might be a good idea to have less branching, but I'm not sure yet, and it'll be minimal either way if it is). Reverted.